### ISSUE TITLE
Can you share an example of using `jaxopt` with `jax.pmap`?

### ISSUE THREAD

**Issue by nasyxx** [google/jaxopt#333]
Can you share an example or a snippets of using `jaxopt` with `jax.pmap`?

When we use optax with jax.pmap, before update parmas, we can use `jax.lax.pmean` to average the gards, so that the params in all of the devices are the same after optax optimizer.  

However, when we use jaxopt, we directly get the params in every devices.  How can we reduce the loss/grads from all devices to the main one so we can update and use the same params in all devices?

**Comment by mblondel** [comment]
We are working on concise notebook examples together with @fllinares. 

In the meantime, you can check out this [example](https://jaxopt.github.io/stable/auto_examples/deep_learning/distributed_flax_imagenet.html#sphx-glr-auto-examples-deep-learning-distributed-flax-imagenet-py).

**Comment by nasyxx** [comment]
> In the meantime, you can check out this [example](https://jaxopt.github.io/stable/auto_examples/deep_learning/distributed_flax_imagenet.html#sphx-glr-auto-examples-deep-learning-distributed-flax-imagenet-py).

Thanks!

**Comment by nasyxx** [comment]
Thanks, again!
